lib/src/image/ops/resize_with_crop_or_pad_op.dart

```diff
@@ -15,11 +16,14 @@ import 'package:tflite_flutter_helper/src/image/tensor_image.dart';
 class ResizeWithCropOrPadOp implements ImageOperator {
   final int _targetHeight;
   final int _targetWidth;
+  final Tuple2<int, int> _cropPosition;
```

Comment:
Can we replace tuple with two separate params.

```
final int _cropLeft;
final int _cropTop;
```

Reply:
We will have to extra check to ensure both are null or non-null at the same time else throw argument error.

Reply:
> 
> 
> Thanks a lot, @mdejeans, for the PR!, Truly sorry to be late, I was inactive for a while.

Don't worry, everyone need some holiday! ;)

Reply:
Done I've removed Tuple deps for user!